NLog.Web.ASPNET5/LayoutRenderers/AspNetCookieLayoutRenderer.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Text;
+#if !DNX
+using System.Web;
+using System.Collections.Specialized;
+#else
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.Extensions.Primitives;
+#endif
+using NLog.LayoutRenderers;
+using System.Collections.Generic;
+using NLog.Config;
+using System;
+
+
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Request Cookie
+    /// </summary>
+    [LayoutRenderer("aspnet-request-cookie")]
+    public class AspNetCookieLayoutRenderer : AspNetLayoutRendererBase
+    {
+        private static string doubleQuotes = "\"";
+        private static string jsonStartBraces = "{";
+        private static string jsonEndBraces = "}";
+        private static string jsonElementSeparator = ",";
+
+        private static string flatCookiesSeparator = "=";
+        private static string flatItemSeperator = ",";
```

Comment:
good practice this (variables instead of magic strings), maybe private 'const' are even better.